components/elements/button.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <button
+    :class="{
+      primary,
+      secondary,
+      tertiary
+    }"
+  >
+    <slot />
+  </button>
+</template>
+
+<script>
+export default {
```

Comment:
Component should export a `name` so tools can display it properly (Ex: devtools)
```suggestion
export default {
  name: 'Button',
```

Reply:
the name is not necessary anymore as vue dev tools and other should take file name by default

Reply:
This is new? The doc haven't been updated yet to reflect that: https://vuejs.org/v2/style-guide/#Priority-B-Rules-Strongly-Recommended-Improving-Readability

And I still had the issue of anonymous component recently 🤔

Name are also very useful in tests - when the component is not registered globally.

Reply:
not it's not new about vue dev tools.

Ok i propose you to create a new pr to update and explain the benefit or not to name component. Usually i name them but yeah it could be great to have a final pr that speak about vue dev tools / jest etc